nemo/collections/nlp/data/datasets/joint_intent_slot_dataset/joint_intent_slot_descriptor.py

```diff
@@ -72,6 +79,9 @@ def __init__(self, data_dir, none_slot_label='O', pad_label=-1):
         self.slots_label_ids = JointIntentSlotDataDesc.label2idx(self.slot_dict_file)
         self.num_slots = len(self.slots_label_ids)
 
+        infold = self.data_dir
+        max_slot_id = 0
+        max_intent_id = 0
```

Comment:
you have access to the slot/intent dicts, are you sure you need this?

Reply:
Used num_slots and num_intents instead.